ansible-ipi-install/roles/node-prep/tasks/10_power_off_cluster_servers.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I  know I got previous feedback - would it be better to install all packages at once instead of breaking out into multiple places

Reply:
Honestly, I only did that for the purposely of clearly seeing why those packages were being installed versus someone asking later why are we installing X package?